components/konflux-ui/OWNERS

```diff
@@ -0,0 +1,19 @@
+# See the OWNERS docs: https://go.k8s.io/owners
+
+approvers:
+- abhinandan13jan
+- caugello
+- CryptoRodeo
+- JoaoPedroPP
+- Katka92
+- sahil143
+- testcara
+
+reviewers:
```

Comment:
Should we add djanaki and rakshett as reviewers

Reply:
This is for deployment only